### DISCUSSION TITLE
Plugin using aliases in the db view

### DISCUSSION THREAD

**Discussion by isle9** [RafaelGB/obsidian-db-folder#185] [Q&A]
![image](https://user-images.githubusercontent.com/74906541/177034194-2d4fbb3e-8be0-4084-bb18-a8812e9bb387.png)
Why exactly does this happen?
In the file itself, there's no alias.

It's not that bad when you use text fields since it's hidden, but in this format, it's pretty ugly and cluttering.

**Comment by RafaelGB** [comment]
With that format evicts conflicts with files that have the same name. The Wikilink is the path and the alias is the filename

**Reply by isle9** [reply]
Unfortunately, this makes manually changing that data in the files a lot harder + it creates double tags in the db view since I use normal links when I link inside files.

Can't this be integrated somehow with the native Obsidian linker? 
And/or, can't this only be used for files that already have duplicates instead of just using this format on everything?
IIRC Obsidian just adds the full path if the name is used by more than one file.

**Reply by RafaelGB** [reply]
In the future we can find another way to manage the links, sure, but now requires too much work. We can try it in some few months

**Reply by isle9** [reply]
Sure, sure. In the meantime, I'll probably just switch to text field after link auto-completion is implemented.
